### PR TITLE
604 implement data-picker

### PR DESCRIPTION
**Pull request by anechytailo**
Pull Request Template
 

### Type of Pull Request *
- [ ] CHANGE (fix or feature that would cause existing functionality to not work as expected)
- [x] FEATURE (non-breaking change which adds functionality)
- [ ] BUGFIX (non-breaking change which fixes an issue)
- [ ] ENHANCEMENT  (non-breaking change which improves existing functionality)
- [ ] NONE (if none of the other choices apply. For example, tooling, build system, CI, docs, etc.)

### Related links
 
**Issue link**
 
**Story link**
[#604 Story](https://github.com/ita-social-projects/dokazovi-requirements/issues/604)
 
### Description *
Create Date and time picker to schedule publish time.
 
### Summary of change

 
"*" - mandatory field

### CODE REVIEW

File: src/styles/theme/index.ts [hunk 1]
```diff
@@ -26,5 +26,10 @@ export const MAIN_THEME = createMuiTheme({
         textTransform: 'none',
       },
     },
+    MuiIconButton: {
+      root: {
+        color: 'initial',
```
Comment (skyboyer) [inline src/styles/theme/index.ts:31]: 👍 👍 👍

File: src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx [hunk 2]
```diff
@@ -43,13 +43,13 @@ const ChangePublicationDateModal: React.FC<IChangeViewsCountModal> = ({
     >
       <Typography>{t(langTokens.admin.currentPublicationDate)}</Typography>
       <Typography className={classes.currentDate}>
-        {posts[id].createdAt}
+        {posts[id].publishedAt}
       </Typography>
       <Typography>{t(langTokens.admin.newPublicationDate)}</Typography>
       <DatePicker
-        value={newPostPublicationDate.split('.').reverse().join('-')}
+        value={new Date(newPostPublicationDate)}
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/ChangePublicationDateModal.tsx:50]: 👍

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      padding: '0 3px',
+      zIndex: '10',
+    },
+  },
+
+  currentDate: {
+    display: 'inline-block',
+    border: 'solid 1px #c4c4c4',
+    borderRadius: '4px',
+    padding: '18.5px 14px',
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx:30]: fractional part in `padding` looks suspicious. Why is it needed here?

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    >
+      <Box className={classes.dateRangeIcon}>{dateRangeIcon}</Box>
+      <DateTimePicker
+        label={t(langTokens.admin.choosePublicationDate)}
+        value={new Date(newPostPublicationDate)}
+        variant="inline"
+        format="dd.MM.yyyy HH:mm"
+        ampm={false}
+        InputProps={{ disableUnderline: true }}
+        // eslint-disable-next-line
```
Comment (skyboyer) [inline src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx:61]: what ESLint is complaining here for?

Comment (anechytailo) [reply src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx:61]: it says props duplicating is forbidden, don't know why ESLint is not case sensitive and doesn't take them as two different props

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx (hunk 3 above)
Comment (anechytailo) [reply src/views/Profile/AdminTable/Modals/SetPublicationDateModal.styles.tsx:30]: I made it the same as it is on other modal forms, but I can take 0.5px away, hope QA won't notice it ))

File: src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx (hunk 4 above)
Comment (skyboyer) [reply src/views/Profile/AdminTable/Modals/SetPublicationDateModal.tsx:61]: https://github.com/jsx-eslint/eslint-plugin-react/issues/2726

please try override `{ignoreCase: false}` for `react/jsx-no-duplicate-props` rule

**Pull request closed without merging**